dataactbroker/handlers/fileHandler.py

```diff
@@ -488,18 +491,25 @@ def call_d_file_api(self, file_type_name, cgac_code, start_date, end_date, job,
         else:
             self.complete_generation(task_key, file_type)
 
-    def download_file(self, local_file_path, file_url):
+    def download_file(self, local_file_path, file_url, upload_name, response):
         """ Download a file locally from the specified URL, returns True if successful """
         if not self.isLocal:
-            with open(local_file_path, "w") as file:
-                # get request
-                response = requests.get(file_url)
-                if response.status_code != 200:
-                    # Could not download the file, return False
-                    return False
-                # write to file
+            bucket = CONFIG_BROKER['aws_bucket']
```

Comment:
in the S3 Handler we have, is there a method that can return the connection to the S3 bucket? if not it may be beneficial to abstract out this part of code there for reuse.